www/src/components/dodont/index.module.scss

```diff
@@ -1,19 +1,80 @@
-.dont:after,
-.dont:before {
-    content: '';
-    position: absolute;
-    top: 50%;
-    left: 50%;
-    -webkit-transform: translateX(-50%) translateY(-50%) rotate(45deg);
-    transform: translateX(-50%) translateY(-50%) rotate(45deg);
-    height: calc(100% + 32px);
-    width: 5px;
-    background: #ff5a5f;
-    opacity: 0.6;
-    border-radius: 2px;
-    z-index: 1;
+@import '~@thumbtack/thumbprint-tokens/dist/scss/_index';
+
+// provides x overlay on top of content region
+.crossout {
+    &:after,
+    &:before {
+        content: '';
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        -webkit-transform: translateX(-50%) translateY(-50%) rotate(45deg);
+        transform: translateX(-50%) translateY(-50%) rotate(45deg);
+        height: calc(100% + 32px);
+        width: 5px;
+        background: #ff5a5f;
+        opacity: 0.6;
+        border-radius: 2px;
+        z-index: 1;
+    }
+    &:before {
+        -webkit-transform: translateX(-50%) translateY(-50%) rotate(135deg);
+        transform: translateX(-50%) translateY(-50%) rotate(135deg);
+    }
 }
-.dont:before {
-    -webkit-transform: translateX(-50%) translateY(-50%) rotate(135deg);
-    transform: translateX(-50%) translateY(-50%) rotate(135deg);
+
+// alternative option that excludes the X overlaying content
+.coloredHeading {
+    &.do {
+        svg {
+            fill: $tp-color__green;
+        }
+
+        > div:last-of-type {
+            border-top: 2px solid $tp-color__green !important;
+        }
+    }
+
+    &.dont {
+        svg {
+            fill: $tp-color__red;
+        }
+        > div:last-of-type {
+            border-top: 2px solid $tp-color__red !important;
+        }
+    }
+}
+
+.isTable {
+    border-collapse: separate;
+
+    td,
+    th {
+        vertical-align: top;
+        border-left: 2px solid $tp-color__white !important;
+        border-collapse: separate !important;
+        border-spacing: 12px !important;
+        padding: 16px !important;
+    }
+
+    th {
+        &:first-of-type {
+            svg {
+                fill: $tp-color__green;
+            }
+
+            > div:last-of-type {
+                border-bottom: 2px solid $tp-color__green !important;
+            }
+        }
+
+        &:last-of-type {
+            svg {
+                fill: $tp-color__red;
+            }
+            > div:last-of-type {
+                border-bottom: 2px solid $tp-color__red !important;
+            }
+        }
+    }
 }
```

Comment:
Try to avoid nesting + element CSS selectors. Use classnames more, when possible.

Reply:
@danoc this should be addressed with the most recent commit.